### PR TITLE
feat(ctp): move Teleportr contracts to periphery

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Copies TeleportrDeposit and TeleportrDisburser to the contracts-periphery
package in preparation for an upcoming audit.

### CODE REVIEW

File: packages/contracts-periphery/contracts/L1/TeleportrDeposit.sol [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+     * @notice Emitted any time a successful deposit is received.
+     *
+     * @param depositId A unique sequencer number identifying the deposit.
+     * @param emitter   The sending address of the payer.
+     * @param amount    The amount deposited by the payer.
+     */
+    event EtherReceived(uint256 indexed depositId, address indexed emitter, uint256 indexed amount);
+
+    /**
+     * @custom:semver 0.0.1
```
Comment (tynes) [inline packages/contracts-periphery/contracts/L1/TeleportrDeposit.sol:77]: In my mind i tied the semver tag to contracts that inherit from `Semver`

**Comment by tynes** [comment]
No delete from `packages/contracts`?

**Comment by smartcontracts** [comment]
> No delete from `packages/contracts`?

Not yet. IMO better to keep in both places for now until we ship bedrock and can delete all the old contracts.

**Comment by tynes** [comment]
Not sure what would cause this to be honest:

```
test/nft-bridge.spec.ts:7:38 - error TS2307: Cannot find module '@eth-optimism/contracts-periphery/artifacts/contracts/L1/messaging/L1ERC721Bridge.sol/L1ERC721Bridge.json' or its corresponding type declarations.

7 import Artifact__L1ERC721Bridge from '@eth-optimism/contracts-periphery/artifacts/contracts/L1/messaging/L1ERC721Bridge.sol/L1ERC721Bridge.json'
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
test/nft-bridge.spec.ts:8:38 - error TS2307: Cannot find module '@eth-optimism/contracts-periphery/artifacts/contracts/L2/messaging/L2ERC721Bridge.sol/L2ERC721Bridge.json' or its corresponding type declarations.

8 import Artifact__L2ERC721Bridge from '@eth-optimism/contracts-periphery/artifacts/contracts/L2/messaging/L2ERC721Bridge.sol/L2ERC721Bridge.json'
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

ERROR: 1

Exited with code exit status 1
```

**Review by tynes — APPROVED** [review]

**Review by maurelian — APPROVED** [review]

**Pull request merged**